deploy/helm-chart/values.yaml

```diff
@@ -0,0 +1,23 @@
+apiToken:
+paasToken:
+operator:
```

Comment:
This is misleading as settings below this node are solely used for a _OneAgent_ instance. Child nodes should be on the root level.